pkg/scalers/gcp_pub_sub_scaler.go

```diff
@@ -244,7 +244,7 @@ func (s *pubsubScaler) getMetrics(ctx context.Context, metricType string) (int64
 func getSubscriptionData(s *pubsubScaler) (string, string) {
 	var subscriptionID string
 	var projectID string
-	regexpExpression, _ := regexp.Compile(compositeSubscriptionIDPrefix)
+	regexpExpression := regexp.MustCompile(compositeSubscriptionIDPrefix)
```

Comment:
Should we move this from here to a package scoped variable?

Reply:
Good point, fixed.